infrastructure/docs/docker/docs/docker-aws.md

```diff
@@ -11,29 +11,120 @@ AWS has the possibility of directly building a Docker image, allowing us to just
         aws_secret_access_key = your_access_key_secret
     ```
 
-3. Create a Dockerrun.aws.json file in the root directory of your application. It should look like this:
-    ```bash
+### Create a Dockerrun.aws.json file
+
+#### Configuring only one container in the instance
+
+AWS has the possibility of directly building a Docker image, allowing us to just push our code (with a [valid Dockerfile](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_docker_image.html#create_deploy_docker_image_dockerfile)). Using this Dockerrun.aws.json, Amazon will build the image in the instance being deployed.
+
+Create a Dockerrun.aws.json file in the root directory of your application.
+
+```bash
+    {
+    "AWSEBDockerrunVersion": "1",
+    "Ports": [
         {
-        "AWSEBDockerrunVersion": "1",
-        "Ports": [
-            {
-            "ContainerPort": "80"
-            }
-        ],
-        "Volumes": [],
-        "Logging": "/var/log/nginx"
+        "ContainerPort": "80"
         }
-    ```
+    ],
+    "Volumes": [],
+    "Logging": "/var/log/nginx"
+    }
+```
+
+#### Configuring more than one container in the instance
+
+For some applications it might be necessary to deploy more than one container to the instance. For example:
+
+    * Ruby on Rails app that needs to run the server, sidekiq and redis
+    * NodeJS app that needs redis or any other service
+
+In such cases, AWS provides the possibility to deploy more than one container to the same instance. If using this option, AWS will no build the docker images for us. To get this working, you'll need to follow these steps:
+
+    1. Visit [AWS ECS](https://console.aws.amazon.com/ecs/home) and create a new repository for the app image.
+    2. Obtain the ElasticBeanstalk instance role in `ElasticBeanstalk` -> `Configuration` -> `Security` -> `Service Role`
+    3. Give the ElasticBeanstalk server permissions to access this repository by running: `aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly --role-name <instance_role_from_step_2> --profile profile_name`
+    4. In the repository root, run the following commands:
+
+```bash
+    aws ecr get-login --no-include-email --profile profile_name
+    docker build -t my-app .
+    docker tag my-app:latest <aws_ecs_repository>/my-app:latest
+    docker push <aws_ecs_repository>/my-app:latest
```

Comment:
I assume we need to do this (build, tag and push) every time we want to deploy a new version. I would add a note at the end explaining this.

Reply:
👍